doc/source/parameters/cfd/boundary_conditions_multiphysics.rst

```diff
@@ -37,6 +37,13 @@ The default parameters for ``temperature`` and ``convection-radiation`` are show
         set Function expression = 0
       end
     end
+    subsection bc 2
+      set id     = 2
+      set type   = heat-flux
+      subsection q0
+        set Function expression = 0
+      end
```

Comment:
I'm not sure q0 is the right name here. why not just heat-flux?

Reply:
I just changed it to "value", to follow what is being used for the Dirichlet BC.